src/components/NearMe/NearMeProfiles.tsx

```diff
@@ -0,0 +1,108 @@
+import React, { useEffect, useState } from 'react'
+import { Box, ImageList, ImageListItem, Typography } from '@mui/material'
+import { LocationOn } from '@mui/icons-material'
+import { makeStyles } from 'tss-react/mui'
+
+type isMobileProps = {
+  isMobile: boolean
+}
+type objectType = {
+  id: string
+  location: string
+  picture: string
+  fName: string
+  lName: string
+}
+const NearMeProfiles = ({ isMobile }: isMobileProps) => {
+  const { classes } = useStyles()
+  const [list, setList] = useState<Array<objectType>>([])
+  const columns = isMobile ? 3 : 6
+  const mrgLeft = isMobile ? 1 : 5
+  const txtAlign = isMobile ? 'center' : 'left'
+  const getData = () => {
+    fetch('users.json', {
+      headers: {
+        'Content-Type': 'application/json',
+        Accept: 'application/json',
+      },
+    })
+      .then(function (response) {
+        return response.json()
+      })
+      .then(function (myJson) {
+        setList(myJson)
+      })
+      .catch((e) => {
+        console.log(e.message)
+      })
+  }
+  useEffect(() => {
+    getData()
+  }, [])
+  return (
+    <Box marginLeft={mrgLeft}>
+      <Typography className={classes.headerNear} sx={{ textAlign: txtAlign }}>
+        Near by
+      </Typography>
+      <Typography className={classes.description} sx={{ textAlign: txtAlign }}>
+        These people near you – just like them and see if it’s a match!
+      </Typography>
+      <Box p={2}>
+        <ImageList cols={columns}>
+          {list
```

Comment:
all such manipulation should be not in the html part. Move plz to separate functions with meaningful names, maybe even to helper file with

Reply:
The layout has been refactored to use Grid instead of Box.

Reply:
Was not resolved (